server/src/main/java/org/cloudfoundry/identity/uaa/health/HealthzEndpoint.java

```diff
@@ -45,8 +54,28 @@ public String getHealthz(HttpServletResponse response) {
             response.setStatus(503);
             return "stopping\n";
         } else {
-            return "ok\n";
+            if (wasLastConnectionSuccessful == null) {
+                return "UAA running. Database status unknown.\n";
+            }
+
+            if (wasLastConnectionSuccessful) {
+                return "ok\n";
```

Comment:
This should indicate database connection is successful, especially from a searching logs/responses.

Reply:
Initially I left this message the same so anyone checking on the current message would not see a difference. 
But per your request I extended the message, mentioning the DB connection was successful.